lib/compare_linker.rb

```diff
@@ -62,11 +69,17 @@ def make_compare_links
   end
 
   def add_comment(repo_full_name, pr_number, compare_links)
-    res = octokit.add_comment(
+    res = requested_repository_octokit.add_comment(
       repo_full_name,
       pr_number,
       compare_links
     )
-    "https://github.com/#{repo_full_name}/pull/#{pr_number}#issuecomment-#{res.id}"
+    res[:html_url]
```

Comment:
Is this the intended change?

Reply:
Yes.
This is because in case of response from github enterprise, the URL is not github.com.

Also, even if it is not github enterprise, I thought that using html_url included in the response is natural.
There is no change in behavior.